defaults/main.yml

```diff
@@ -5,20 +5,26 @@
 consul_debug: no
 
 ### Package
-consul_version: "{{ lookup('env','CONSUL_VERSION') | default('0.8.2', true) }}"
+consul_version: "{{ lookup('env','CONSUL_VERSION') | default('0.8.3', true) }}"
 consul_architecture_map:
   # this first entry seems... redundant
   #(but it's required for reasons)
   amd64: amd64
   x86_64: amd64
   armv7l: arm
   aarch64: arm64
+  32-bit: "386"
+  64-bit: amd64
 consul_architecture: "{{ consul_architecture_map[ansible_architecture] }}"
-consul_os: "{{ ansible_system | lower }}"
+consul_os: "\
+  {% if ansible_os_family == "Windows" %}\
+    {{ 'windows' }}\
+  {% else %}\
+    {{ ansible_system | lower }}\
```

Comment:
`ansible_system` in these cases returns **Win32NT**. This changes between Windows Server versions and is unfortunately brittle.